src/components/Remark.tsx

```diff
@@ -67,20 +73,20 @@ const Remark = ({ className, targetChain }: Props) => {
     const estimatedFee = estimatedFeeType.toString();
 
     const bridgeMessage = sourceApi.tx[`bridge${targetChain}MessageLane`].sendMessage(lane_id, payload, estimatedFee);
-
-    const nonceCall = await sourceApi.rpc.system.accountNextIndex(account.address);
-    const nonce = nonceCall.toNumber();
-    await bridgeMessage.signAndSend(account, { nonce });
+    await bridgeMessage.signAndSend(account, { nonce: -1 });
+    setIsExecuting(false);
```

Comment:
`try-catch-finally`?

Reply:
oh i forgot this one... committing it soon